src/test/kotlin/no/nav/syfo/api/PersistenceUtilsSpek.kt

```diff
@@ -43,42 +46,55 @@ object PersistHendelseSpek : Spek({
         false,
         "1234",
         "apen-isyfo-stoppautomatikk",
-        0L
+        5000L
```

Comment:
👀

Reply:
Er ikke sikkert vi trenger denne lenger, alt kan bare vært 0 overalt hele tiden

Reply:
Oops, denne var for å teste litt 😅